full/pom.xml

```diff
@@ -157,6 +157,16 @@ under the License.
 		</dependency>
 		<dependency>
 			<groupId>org.apache.metamodel</groupId>
+			<artifactId>MetaModel-neo4j</artifactId>
+			<version>${project.version}</version>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.metamodel</groupId>
+			<artifactId>MetaModel-json</artifactId>
```

Comment:
Eh, what? Is MM-json missing in full module? If so that's an independent bug and something to add separately I think.